.github/workflows/pkgci_regression_test.yml

```diff
@@ -139,17 +139,21 @@ jobs:
           # CPU
           - name: cpu_llvm_task
             models-config-file: models_cpu_llvm_task.json
+            iree-tests-cache: ~/iree_tests_cache
             runs-on: nodai-amdgpu-w7900-x86-64
 
           # AMD GPU
           - name: amdgpu_rocm_mi250_gfx90a
             models-config-file: models_gpu_rocm_gfx90a.json
+            iree-tests-cache: /groups/aig_sharks/iree-tests-cache
```

Comment:
This is fine for the current migration, but we could decouple the cache location from the workflow file so this could vary between machines and we can update it independently from the workflow file sources.

As-is, we have jobs using the old cache location trying to download files and running out of space. Even if we merge this, there will still be some time when PRs synced to older commits will use that old location.

Maybe set an environment variable on the runner itself, or put the runner software in a docker container that has things like that (and available GPUs) configured.

Reply:
Merging as that queue of unsynced PRs is building up